project/src/main/webapp/script.js

```diff
@@ -40,13 +40,24 @@ function fetchFromQuery() {
   const placesDiv = document.getElementById('place');
   displayResultsPage();
   const map = createMap();
-  fetch('/query?' + params).then(response => response.json()).then((places) => {
-    places.forEach((singlePlace) => {
-      placesDiv.appendChild(createPlaceElement(singlePlace));
-      addPlaceMarker(map, singlePlace)
-    });
-    displayAfterResults();
-  });
+  fetch('/query?' + params)
+      .then(response => response.json())
+      .then((places) => {
+        places.forEach((singlePlace) => {
+          placesDiv.appendChild(createPlaceElement(singlePlace));
+          addPlaceMarker(map, singlePlace)
+        });
+        displayAfterResults();
+        if (places.length < 3) {
```

Comment:
It is great that you added this handeling. I just want to bring to attention that it doesn't match what we planned in the [design](https://docs.google.com/document/d/1yPTb2Rh6MSTrDE9b9e1Q61ZCcH_cVNqBI8qs1VKNZEs/edit#bookmark=id.hr310z4jfen1).
I am fine with not doing what we said regarding the zero results because it may complicate things and we don't have time for that, but then we should update the design.

Reply:
Yeah, as we recently decided (and updated in the Engineering Requirements part), the more graceful solution will be implemented in P2. This was not updated in the Product Requirements so I updated it accordingly.